components/Home/Leaderboard/index.jsx

```diff
@@ -1,52 +1,76 @@
 import { useState, useEffect } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
-import Link from 'next/link';
-import { Typography, Table } from 'antd/lib';
-import { LinkOutlined } from '@ant-design/icons';
-
+import {
+  Typography, Table, Button, Modal, List, Alert,
+} from 'antd/lib';
 import { setLeaderboard } from 'store/setup/actions';
 import { getLeaderboardList } from 'common-util/api';
-import { DOCS_SECTIONS } from 'components/Documentation/helpers';
+import { EducationTitle } from 'components/Education';
 import { DiscordLink } from '../common';
 import { LeaderboardContent } from './styles';
 
 const { Title, Text } = Typography;
 
-const columns = [
-  { title: 'Rank', dataIndex: 'rank' },
-  { title: 'Name', dataIndex: 'name' },
-  { title: 'Points Earned', dataIndex: 'points' },
-];
-
-const Actions = () => (
-  <>
-    <Title level={2}>Actions</Title>
-    <Text type="secondary" className="custom-text-secondary">
-      Complete actions to earn points, climb the leaderboard and upgrade your
-      badge.&nbsp;
-      <Link href={`/docs#${DOCS_SECTIONS.actions}`}>Learn more</Link>
-    </Text>
-
-    <Text style={{ display: 'block' }}>
-      <a
-        href="https://discord.com/channels/899649805582737479/1030087446882418688/1034340826718937159"
-        target="_blank"
-        rel="noreferrer"
-      >
-        See all actions&nbsp;
-        <LinkOutlined />
-      </a>
-    </Text>
-  </>
-);
-
 const Leaderboard = () => {
   const [isLoading, setIsLoading] = useState(false);
+  const [isModalVisible, setIsModalVisible] = useState(false);
+  const [currentWallet, setCurrentWallet] = useState(null);
+
   const chainId = useSelector((state) => state?.setup?.chainId);
+  const account = useSelector((state) => state?.setup?.account);
   const isVerified = useSelector((state) => state?.setup?.isVerified);
   const data = useSelector((state) => state?.setup?.leaderboard);
   const dispatch = useDispatch();
 
+  const handleOpenModal = (wallet) => {
+    setCurrentWallet(wallet);
+    setIsModalVisible(true);
+  };
+
+  const handleCreateTweet = () => {
+    window.open(
+      `https://twitter.com/intent/tweet?text=I'm linking my wallet to @Autonolas Contribute:%0A${currentWallet}%0A%0ALearn more—${encodeURIComponent(
```

Comment:
@dvilelaf note that this was the Twitter message I went for – the other one felt really convoluted from a marketing perspective.

Reply:
Ok. The link should be a link to the whitepaper, not one to the main site.

Reply:
Although we can change that in the backend oc